modules/default/calendar/calendarfetcherutils.js

```diff
@@ -16,7 +16,7 @@ const CalendarFetcherUtils = {
 	 * until: the date until the event should be excluded.
 	 */
 	shouldEventBeExcluded (config, title) {
-		let filter = {
+		let result = { // change name
```

Comment:
Are the comments really helpful? I find them rather confusing, as they seem to refer to the PR rather than functionality.